nifi-assembly/pom.xml

```diff
@@ -983,6 +983,9 @@ language governing permissions and limitations under the License. -->
             <id>include-graph</id>
             <activation>
                 <activeByDefault>false</activeByDefault>
+                <property>
+                    <name>allProfiles</name>
```

Comment:
Could `allProfiles` be set as a peppery as part of a new profile, e.g. a profile called `include-all` that just contains a

```xml
<properties>
  <allProfiles>true</allProfiles>
</properties>
```

Block? That way we can still use the `mvn -p` profile flag instead of `-D` setting, which may be a bit more obvious for people

I haven't tried this though so don't know whether the profiles dependency/linking would work